paddlenlp/ops/patches/FasterTransformer/cuda/topk_kernels.cu

```diff
@@ -553,7 +595,7 @@ void topK_sampling_kernel_kernelLauncher(void* workspace,
                                          DecodingSamplingArguments args,
                                          cudaStream_t stream) {
   std::minstd_rand engine;
-  int seed = std::random_device()();
+  int seed = 123;
```

Comment:
Why 123 here？ Seems too tricky here.

Reply:
This pr is still working in progress. The seed is for testing.

Reply:
Done